packages/sdks/src/blocks/columns/columns.lite.tsx

```diff
@@ -74,24 +80,55 @@ export default function Columns(props: ColumnProps) {
         '--column-margin-left-tablet': state.maybeApplyForTablet(marginLeft),
       };
     },
+
+    getWidthForBreakpointSize(size: SizeName) {
+      const breakpointSizes = getSizesForBreakpoints(
+        props.customBreakpoints || {}
+      );
+
+      return breakpointSizes[size].max;
+    },
+
+    get columnsStyles(): string {
```

Comment:
can you add a comment with some info on why we moved this from `css` to an interpolated string `styles`

Reply:
sure, will do

Reply:
done, @ln #130